<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ctOpenglWidget" name="openGLWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>实时流地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Url"/>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_TCP">
           <property name="text">
            <string>TCP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_UDP">
           <property name="text">
            <string>UDP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play">
           <property name="text">
            <string>播放</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>本地文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_File"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_open">
           <property name="text">
            <string>打开文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play2">
           <property name="text">
            <string>播放</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>相关操作：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_stop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_record">
           <property name="text">
            <string>录像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_stop_record">
           <property name="text">
            <string>停止录像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_snapshot">
           <property name="text">
            <string>截图</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_open_audio">
           <property name="text">
            <string>打开音频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close_audio">
           <property name="text">
            <string>关闭音频</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctOpenglWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>ctopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
